### PR TITLE
Update best-practices-for-collections.md

### PR DESCRIPTION
**Pull request by JordanTheITGuy**
# Summarize the change in the pull request title

Updated times for the maintenance windows. 
When a patch applies it takes the maximum run time + five minutes, adds that to the current time to see if 65 minutes from now is after the maintenance window ends. 

General best practice in the community for years has been to use at least 4 hours as the default tome.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger8** [comment]
@JordanTheITGuy : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
Hi @JordanTheITGuy, thanks for the contribution. The challenge with this guidance is that maintenance windows could vary due to other factors. While the default runtime is 60 minutes, some customers do change these based on testing. And that runtime can vary depending upon the specific hardware and specific configuration. So I think it's tough to give explicitly guidance for the length of the maintenance window. I think we could reword this section to get into some of the nuances that they should consider when determining the length of their window, for example:
- default (60) vs custom runtime on update
- make sure window is longer than max runtime
- 5 minutes reserved for reboot
Basically, just stick to the facts, give the reader the info that they can decide for themselves (mixed with their own business requirements). 

So...if you want to rework this PR to address that, great. Otherwise, we can close this PR, and file an issue for us to address in the future. 😄

**Comment by JordanTheITGuy** [comment]
Good morning! Apparently the e-mail send didnt' seem to update. If you would like to kick it back to me I'll be happy to work on re-wording and or maybe just adding a "note" block that adds some of the information that we've discussed here that helps cover those points. 😊

**Comment by PRMerger9** [comment]
@aczechowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@aczechowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
#sign-off 

*Once merged*, this should be live with the next daily publishing process. Right now that *should* be today (Thursday, 26 September) around 11:00 Pacific time. No further action needed from you.
